packages/react-dom/src/events/plugins/ModernEnterLeaveEventPlugin.js

```diff
@@ -126,16 +126,16 @@ function extractEvents(
 
   if (topLevelType === TOP_MOUSE_OUT || topLevelType === TOP_MOUSE_OVER) {
     eventInterface = SyntheticMouseEvent;
```

Comment:
Could we maybe hoist the strings out to variables? It might help code size non gzip.

Reply:
Maybe! I want to do a few more simplifying passes before optimizing.